src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -181,6 +182,56 @@ public static void CopyToShorterArray()
             Assert.Equal<int>(expected, dst);  // CopyTo() checks for sufficient space before doing any copying.
         }
 
+        private static void CopyTo_LargeSize_TestHelper(long BufferSize)
```

Comment:
nit: Rename and remove the underscores. CopToLargeSizeTestHelper follows the naming guidelines better.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/framework-design-guidelines-digest.md#naming-guidelines